webapp/view/ListReport.view.xml

```diff
@@ -0,0 +1,187 @@
+<mvc:View
+	xmlns="sap.m"
+	xmlns:mvc="sap.ui.core.mvc"
+	xmlns:f="sap.f"
+	xmlns:core="sap.ui.core"
+	xmlns:fb="sap.ui.comp.filterbar"
+	controllerName="products.app.controller.ListReport">
+
+		<f:DynamicPage 
+			id="dynamicPageId" 
+			preserveHeaderStateOnScroll="true" 
+			showFooter="true" 
+			fitContent="true">
+			<!-- DynamicPage Title -->
+			<f:title>
+				<f:DynamicPageTitle>
+					<f:heading>
+						<Title text="{i18n>ListReportTitle}"/>
+					</f:heading>
+					
+				</f:DynamicPageTitle>
+			</f:title>
+
+			<!-- DynamicPage Header -->
+			<f:header>
+				<f:DynamicPageHeader>
+					
+					<f:content>
+						
+						<fb:FilterBar id="filterbar" persistencyKey="myPersKey" useToolbar="false" >
+							<fb:filterGroupItems >
+
+								<fb:FilterGroupItem 
+									name="search" 
+									label="{i18n>NameText} {i18n>AndText} {i18n>DescriptionText}" 
+									groupTitle="false" 
+									groupName="HeaderFiltersGroup" 
+									visibleInFilterBar="true">
+									<fb:control>
+										<SearchField 
+											id="searcher" 
+											width="100%" 
+											liveChange="onFilter(null)" 	
+										/>
+									</fb:control>
+								</fb:FilterGroupItem>
+
+								<fb:FilterGroupItem 
+									name="Availability" 
+									label="{i18n>AvailabilityText}" 
+									groupName="HeaderFiltersGroup" 
+									visibleInFilterBar="true">
+									<fb:control>
+										<Select
+											id="availabilitySelect"
+											forceSelection="false"
+											selectedKey="all"
+											change="onFilter(null)"
+										>
+										<core:Item key="all" text="all" />
```

Comment:
Please remove all. Let user select all or nothing in order to see all products in the table

Reply:
I don't understand a bit, I just should delete text="all" or I could leave it, but with i18n?

Reply:
Or you mean that user could press on empty field and in table will be all products?

Reply:
> I don't understand a bit, I just should delete text="all" or I could leave it, but with i18n?

Yes, remove 'all'. There should be only 2 items in select control. If select is empty, show all data in table, if all items are selected, show data in table as well.